### DISCUSSION TITLE
APmodules unavailable as overlayFS is disabled by kernel

### DISCUSSION THREAD

**Discussion by mnojz** [bmax121/APatch#193] [Q&A] [answered]
after i root my phone using apatch app, i was trying to flash some magisk modules like youtube revanced but when i goto **APmodules** i couldn't find **load** option and there is **"modules are unavailable as overlayfs is disabled by kernel"** message shown. 
is this issue resolvable or there is limitation on my phone _( Redmi 5a )_.  or am i doing something wrong.

**how i root my phone**
================

1. i installed apatch app from github
2. i extracted boot.img file from stock ROM that is recently flashed on my phone.
3. i opened apatch app then set key for it. then i patch boot.img file and i get another _apatch_10474_dbxq_boot.img_ file on downloads folder 
4. i copied _apatch_10474_dbxq_boot.img_ file into my pc.
5. i reboot my phone into fastboot mode and ran this command **fastboot flash boot apatch_10474_dbxq_boot.img**
6. after rebooting phone, i opened apatch app and installed **androidPatch**
7. both **KernelPatch**  and **AndroidPatch** are working 

but APmodules seems not working 
![Screenshot 2024-01-31 163434](https://github.com/bmax121/APatch/assets/78907808/5ce6b625-fdc1-41cd-984a-4d7f1211e322)

**Comment by marat2509** [comment, marked as answer]
OverlayFS is not supported by your kernel. This can only be solved by recompiling it.

**Comment by Sahil71684** [comment]
Can you please add a toggle to enable all Apps on superuser and go lock it in. Right now you have to enable if one by one. And after reboot it resets And you have to enable all one by one again. Please. Thank you for your awesome work.
